packages/docusaurus-search-local/babel.config.js

```diff
@@ -1,11 +1,7 @@
-const assert = require("assert");
-
 module.exports = (api) => {
-  const isTest = api.env("test");
-  assert(isTest, "babel.config.js is only used for testing with jest!");
+  api.cache.never();
```

Comment:
Hey

Just curious, do you still need this file and why?
This is going to be incompatible with Docusaurus Faster (Rspack/SWC)